content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,99 @@
-======================================
-Taking stock from different warehouses
-======================================
+===========================================================
+Sell Stock from Multiple Warehouses Using Virtual Locations
+===========================================================
 
-When you plan to deliver a customer, you don’t know in advance if the products will come from
-Warehouse A or Warehouse B. You may, in some cases, need to take stock from different warehouses.
-With *Odoo*, you can configure this by using the concept of virtual warehouses. Let us show you how
-to set those virtual warehouses.
+While keeping stock and selling inventory from one warehouse might work for smaller companies,
+bigger companies might need to keep stock in and sell from multiple warehouses in multiple
+locations.
 
-Set up virtual warehouses
-=========================
+Sometimes, products included in a single sales order might take stock from two or more warehouses.
+In Odoo, this can be done by using **Virtual Locations**.
 
-Let’s say you have two warehouses: Warehouse A and Warehouse B. Create a new warehouse, that will be
-a virtual one. It will allow you to take the stock from A or B. To do so, go to your inventory app
-settings and enable the multi-warehouses feature. Then, go to the warehouses menu and click on
-create.
+.. note::
+   To create virtual locations in warehouses, :guilabel:`Storage Locations` will need to be
+   enabled. To do so, go to :menuselection:`Inventory --> Configuration --> Settings`, scroll down
+   to the **Warehouse** section, click the checkbox next to :guilabel:`Storage Locations`, and
+   :guilabel:`Save` changes.
+
+Create and configure a virtual parent location
+==============================================
+
+Before creating any virtual stock locations, a new warehouse will need to be created that will act
+as a virtual warehouse, and will be the parent location of other physical warehouses.
+
+Create a new warehouse
+----------------------
 
-.. image:: stock_warehouses/ware1.png
+To create a new warehouse, go to :menuselection:`Inventory --> Configuration --> Warehouses`, and
+click :guilabel:`New`. From here, the warehouse name can be changed, as well as other details about
+how to resupply the warehouse. Routes can also be set by clicking on the :guilabel:`Routes` smart
```

Comment:
From here, the warehouse name can be changed, and other details about
how to resupply the warehouse can be configured, as well.